h5ai-nginx.conf.template

```diff
@@ -16,6 +16,12 @@ server {
 		try_files $uri @redirect;
 	}
 
+	# When specifically asking for a $uri that ends with `.something` redirect the request to the /server/repository/ API
+	location ~ ^/contracts/(.*\.\w+)$ {
+		rewrite ^/contracts/(.*\.\w+)$ /server/repository/contracts/$1 break;
```

Comment:
`/server` is part of the server root URL.

e.g. on prod. it's `sourcify.dev/server/` but locally it is `localhost:5555/`

Instead change the `SERVER_URL` to `sourcify.dev/server/`
```suggestion
		rewrite ^/contracts/(.*\.\w+)$ /repository/contracts/$1 break;
```

Reply:
I tried it already while doing this PR, but unfortunately the parameter passed to `proxy_pass` is a host, not a host+url. So it cannot be done in this way. If it's necessary I can work on finding a different solution